File: PrototypeWithAuth/Models/VendorContact.cs
using Org.BouncyCastle.Asn1.Mozilla;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PrototypeWithAuth.Models
{
    public class VendorContact
    {
        [Key]
        public int VendorContactID { get; set; }
        [Required]
        [Display(Name = "Contact Name")]
        public string VendorContactName { get; set; }
        [Required]
        [EmailAddress]
        [Display(Name = "Contact Email")]
        public string VendorContactEmail { get; set; }
        [Required]
        [Phone]
        [Display(Name = "Contact Phone")]
        public string VendorContactPhone { get; set; }
        public int VendorID { get; set; }
        public Vendor Vendor { get; set; }

    }
}

File: PrototypeWithAuth/wwwroot/js/VendorContact.js


$(function () {
	$(".contact-name").rules("add", "required");

	$(".contact-email").rules("add", {
				required: true,
				email: true
			});
	$(".contact-phone").rules("add", {
				required: true,
				minlength: 9
			});

});